Source/PluginEditor.cpp

```diff
@@ -108,58 +109,67 @@ void SubsynthAudioProcessorEditor::filterChanged()
 
 //==============================================================================
 void SubsynthAudioProcessorEditor::paint (juce::Graphics& g)
-{
+{   
+    // dynamically obtain Width for resizing purposes
+    width = getWidth();
```

Comment:
You don't need to call this here, just in `resized()` - because we set the width of the window when the plug-in is first booted, it sets everything correctly at first. Then, since width is adjusted every time `resized()` is called, these will scale accordingly.

Reply:
That makes sense - I'll take it out.